docs/_sources/new_bandit.rst.txt

```diff
@@ -262,9 +262,9 @@ Preparing a pull request typically involves the following steps:
 
 a. Add a note about your changes in the CHANGELOG.txt.
 b. Update the library version. You can use a keyword search for "version" to make sure you cover all version fields.
-c. Update the README.md, in necessary.
+c. Update the README.md, if necessary.
 d. Update the documentation rst files under the /docsrc folder , if necessary.
-e. If you update any documentation, make sure to recompile the docs by running ``make github`` under the /docsrc folder.
+e. If you update any documentation, make sure to recompile the docs by running ``make github`` under the /docsrc folder. If you have mabwiser installed, you will need to uninstall it, as Sphinx first looks for an installed version before using the local package.
```

Comment:
quick q: if we make an update to doc files do we need to uninstall the library and then run Sphinx?

Reply:
It might be easier to make sure that mabwiser is installed in editable/dev mode (`pip install - e. `) so that the local package and the installed version are the same. That's what I do for my TextWiser dev environment.

Reply:
@dorukkilitcioglu good to know! I spent an hour last week trying to figure out why the docs weren't updating until I remembered this issue.